Bottom: 29a54d7394f1512ae39f5d6322b2bb0d689ce00c
Top:    1010b23199c55904432edf0d944b316ad242ce3a
Author: Ian Kent <raven@themaw.net>
Date:   2013-02-27 12:43:32 +0800

Refresh of autofs-5.0.7-add-symlink-pseudo-option.patch

---

diff --git a/lib/master_parse.y b/lib/master_parse.y
index f3e0433..11caf5b 100644
--- a/lib/master_parse.y
+++ b/lib/master_parse.y
@@ -57,7 +57,7 @@ static char *type;
 static char *format;
 static long timeout;
 static long negative_timeout;
-static unsigned symlink;
+static unsigned symlnk;
 static unsigned nobind;
 static unsigned ghost;
 extern unsigned global_selection_options;
@@ -559,7 +559,7 @@ option: daemon_option
 
 daemon_option: OPT_TIMEOUT NUMBER { timeout = $2; }
 	| OPT_NTIMEOUT NUMBER { negative_timeout = $2; }
-	| OPT_SYMLINK	{ symlink = 1; }
+	| OPT_SYMLINK	{ symlnk = 1; }
 	| OPT_NOBIND	{ nobind = 1; }
 	| OPT_NOGHOST	{ ghost = 0; }
 	| OPT_GHOST	{ ghost = 1; }
@@ -630,7 +630,7 @@ static void local_init_vars(void)
 	debug = 0;
 	timeout = -1;
 	negative_timeout = 0;
-	symlink = 0;
+	symlnk = 0;
 	nobind = 0;
 	ghost = defaults_get_browse_mode();
 	random_selection = global_selection_options & MOUNT_FLAG_RANDOM_SELECT;
@@ -815,7 +815,7 @@ int master_parse_entry(const char *buffer, unsigned int default_timeout, unsigne
 		entry->ap->flags |= MOUNT_FLAG_RANDOM_SELECT;
 	if (use_weight)
 		entry->ap->flags |= MOUNT_FLAG_USE_WEIGHT_ONLY;
-	if (symlink)
+	if (symlnk)
 		entry->ap->flags |= MOUNT_FLAG_SYMLINK;
 	if (negative_timeout)
 		entry->ap->negative_timeout = negative_timeout;
